app/src/main/java/cm/aptoide/pt/socialMedia/SocialMediaAnalytics.kt

```diff
@@ -0,0 +1,38 @@
+package cm.aptoide.pt.socialMedia
+
+import cm.aptoide.analytics.AnalyticsManager
+import cm.aptoide.analytics.implementation.navigation.NavigationTracker
+import cm.aptoide.aptoideviews.socialmedia.SocialMediaView
+import java.util.*
+
+open class SocialMediaAnalytics(val analyticsManager: AnalyticsManager,
+                                val navigationTracker: NavigationTracker) {
+
+  companion object {
+    @JvmField
```

Comment:
Just curious, are we doing this on purpose? This tells the Kotlin to not  generate getters/setters for this field

Reply:
adding this allows to expose it to the outside. I was wronlgy using this. I should add const val.
Also changed the other ones to val as there was no need for them to be a var